setup.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 by Clearcode <http://clearcode.cc>
+# and associates (see AUTHORS.md).
+
+# This file is part of querystringsafe_base64.
+
+# querystringsafe_base64 is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+
+# querystringsafe_base64 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public License
+# along with querystringsafe_base64. If not, see <http://www.gnu.org/licenses/
+
+import os
+import re
+from setuptools import setup, find_packages
+
+
+here = os.path.dirname(__file__)
+with open(
+    os.path.join(here, 'src', 'querystringsafe_base64', '__init__.py')
+) as v_file:
+    package_version = re.compile(
+        r".*__version__ = '(.*?)'", re.S).match(v_file.read()).group(1)
+
+
+def read(fname):
+    """
+    Read filename.
+    :param str fname: name of a file to read
+    """
+    return open(os.path.join(here, fname)).read()
+
+
+test_requires = [
+    'pytest',
+    'pytest-cov',
+    'pylama',
+]
+
+
+setup(
+    name='querystringsafe_base64',
+    version=package_version,
+    author='Clearcode - The A Room',
+    author_email='thearoom@clearcode.cc',
```

Comment:
I'd do something like `'@'.join(['thearoom', 'clearcode.cc'])` to defend against email harvesters. But maybe my practices are a bit dated.